lib/rubocop/result_cache.rb

```diff
@@ -9,7 +9,7 @@ module RuboCop
   # Provides functionality for caching RuboCop runs.
   # @api private
   class ResultCache
-    NON_CHANGING = %i[color format formatters out debug fail_level
+    NON_CHANGING = %i[color format formatters out debug display_time fail_level
```

Comment:
No new specs because not all other non-changing options have been tested either, [only a small set](https://github.com/rubocop/rubocop/blob/161ba2652849998e65c06023cfa5bb0793f0ebb7/spec/rubocop/result_cache_spec.rb#L271-L294).